.github/ISSUE_TEMPLATE/release-checklist.md

```diff
@@ -5,26 +5,43 @@ Release checklist:
  - [ ] Ensure you can sign commits and any yubikeys/smartcards are plugged in
  - [ ] Run `./tag_release.sh <vX.Y.z> <git commit hash>`
  - [ ] Push that tag to GitHub
- - [ ] Run `./build_releases`
- - [ ] Sign the release artifacts by running
-```
-gpg --local-user 0xCDDE268EBB729EC7! --detach-sign --armor <path to artifact>
-```
-for each release artifact. Do not try to sign all of them at once by globbing. If you do, gpg will sign the combination of all the release artifacts instead of each one individually.
 
- - [ ] Create a draft release on GitHub and upload all the release artifacts and their signatures. Copy and paste the release notes from NEWS here as well.
+Packaging:
+ - [ ] Update the FCCT spec file in [Fedora](https://src.fedoraproject.org/rpms/fedora-coreos-config-transpiler):
+   - Update the commit hash global variable
+   - Bump the `Version`
+   - Switch the `Release` back to `1.git%{shortcommit}%{?dist}`
+   - Remove any patches obsoleted by the new release
+   - Run `go-mods-to-bundled-provides.py | sort` while inside of the fcct directory you ran `./tag_release` from & copy output into spec file in `# Main package provides` section
+   - Add any new spec paths to `%gotest` lines
+   - Update change log
+ - [ ] Download `fcct-$VERSION.tar.gz` from the GitHub tag
```

Comment:
As in https://github.com/coreos/ignition/pull/1119, let's use `spectool` here too: "Run `spectool -g -S ignition.spec`"